imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+class ActiveProfileManager(models.Manager):
+    def get_queryset(self):
+        return super(ActiveProfileManager,
+                     self).get_queryset().filter(user_is_active=True)
```

Comment:
this is going to fail if you ever use it.  There should be **two** underscores between `user` and `is_active`:

```python
    return super(
        ActiveProfileManager, self
    ).get_queryset().filter(user__is_active=True)
```